terraform/variables.tf

```diff
@@ -1,25 +1,26 @@
-variable "project" {
-  description = "Project ID"
-  type        = string
-}
-
-variable "application_name" {
-  description = "Name of the application"
-  type        = string
-}
-
-variable "region" {
-  description = "Region of most of the resources"
-  type        = string
-  default     = "europe-west4"
-}
-
-variable "zone" {
-  description = "Zone of most of the resources"
-  default     = "europe-west4-c"
-}
-
-variable "functions_region" {
-  description = "Region where Cloud functions are deployed."
-  default     = "europe-west1"
+variable "project" {
+  description = "Project ID"
+  type        = string
+}
+
+variable "application_name" {
+  description = "Name of the application"
+  type        = string
+  default = "unnamed"

```

Comment:
Do we really want this here? I think I would prefer to have terraform shout at me that I forgot something rather than find out too late that my resources have "unnamed" in it.

Reply:
Ah I guess this was mainly due to my thinking that usually there isn't just one application name anymore, so figured this was becoming a bit irrelevant

Reply:
But I can remove that

Reply:
It's removed now!